test/test_source_todo.rb

```diff
@@ -137,6 +137,36 @@ def test_todo_colon_parsing_multi_line
     )
   end
 
+  def test_todo_colon_parsing_multi_line_with_empty_line
+    check_valid_puzzle(
+      "
+      // TODO: #45 task description
+      //
+      //  second line after empty line is not a puzzle text
+      ",
+      '2-2',
+      'task description',
+      '45'
+    )
+  end
+
+  # space is needed in test data in the comment
+  # rubocop:disable Layout/TrailingWhitespace
+  def test_todo_colon_parsing_multi_line_with_empty_line_and_space
+    check_valid_puzzle(
+      '
+      // TODO: #46 task description
+      // 
+      //  second line after empty line is a part of the puzzle in case of space exists
```

Comment:
Line 145 also has a leading space after an empty line, but it isn't part of puzzle text. How this line is different?

Reply:
Oh, I see it now: on line 159 there is a trailing space.

Could we have a better rules for distinguish one type of comments from another? The current one seems like a subject for future confusions of our users.

/cc @yegor256 WDYT?